en/docs/tutorials/connect-your-existing-ballerina-project-to-choreo.md

```diff
@@ -0,0 +1,46 @@
+# Connect Your Existing Ballerina Project to Choreo
+
+You can connect your existing Ballerina project repository to Choreo when you create any of the following components:
+
+- REST API
+- Manual Trigger
+- Scheduled Task
+- Webhook
+
+Connecting your existing Ballerina repository to Choreo allows you to keep the source code of a component you create within your control. This capability also allows your development team to work on a particular Choreo component by collaborating via pull requests while adhering to enterprise-specific best practices and development guidelines such as PR checks, code analysis, styling preferences, etc.  
+
+This tutorial walks you through the steps to connect your existing Ballerina project to Choreo when you create a REST API component. 
+
+Let's get started!
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev).
+2. Create a project to add the REST API component. You can follow the instructions under [step 1 in the Connect Your Own GitHub Repository to Choreo](../connect-your-own-github-repository-to-choreo/#step-1-create-a-project-to-add-the-rest-api-component) tutorial.
+3. On the **Components** page, click **+Create**.
+4. Click the **REST API** card.
+5. Click **Existing code**. This displays the REST API dialog box where you can proceed to connect your existing Ballerina project repository to Choreo.
```

Comment:
Should we have a prerequisite mentioning the need to have an existing Ballerina project?

Reply:
Since the topic and the intro paragraphs describe that the tutorial is about connecting a user’s existing Ballerina project repository to Choreo, do we really need a prerequisite mentioning the same?